followthemoney/schema/CallForTenders.yaml

```diff
@@ -0,0 +1,77 @@
+CallForTenders:
+  label: "Call for tenders"
+  plural: "Calls for tenders"
+  description: "A public appeal issued by an authority, possibly on behalf of another, for buying a specific work, supply or service"
+  extends:
+    - Thing
+  required:
+    - title
+  featured:
+    - title
+    - contractingAuthority
+  caption:
+    - title
+  properties:
+    title:
```

Comment:
I think `name` is defined on `Thing`, perhaps that works already?

Reply:
I could use name instead of title (which is the field name of choice from the source)